lib/jsdom/browser/Window.js

```diff
@@ -441,7 +441,10 @@ function Window(options) {
       return customElementRegistry;
     },
     get event() {
-      return idlUtils.wrapperForImpl(window._currentEvent);
+      return window._currentEvent ? idlUtils.wrapperForImpl(window._currentEvent) : undefined;
+    },
+    set event(value) {
+      Object.defineProperty(window, "event", { configurable: true, enumerable: true, writable: true, value });
```

Comment:
<https://github.com/jsdom/jsdom/pull/3137> includes a helper for creating `[Replaceable]` setters.